custom_components/garmin_connect/sensor_descriptions.py

```
@@ -0,0 +1,1110 @@
+
+            **{k: v for k, v in data.get("menstrualData", {}).items()
+               if k not in ("currentPhase",)},
+        },
+    ),
+    GarminConnectSensorEntityDescription(
+        key="menstrualCycleDay",
+        translation_key="menstrual_cycle_day",
+        icon="mdi:calendar-today",
+        state_class=SensorStateClass.MEASUREMENT,
+        value_fn=lambda data: data.get("menstrualData", {}).get("dayOfCycle"),
+        attributes_fn=lambda data: {
```

Ruff (0.14.10)

Unused lambda argument: `data`

(ARG005)